resources/js/features/games/components/GameSetRequests/GameSetRequests.tsx

```diff
@@ -0,0 +1,50 @@
+import type { FC } from 'react';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { GameAvatar } from '@/common/components/GameAvatar';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { PatreonSupporterItem } from '@/features/patreon-supporters/components/PatreonSupporterItem';
+
+export const GameSetRequests: FC = () => {
+  const { game, deferredRequestors, initialRequestors, totalCount } =
+    usePageProps<App.Community.Data.GameSetRequestsPageProps>();
+  const { t } = useTranslation();
+
+  // Combine initial and deferred requestors once deferred props finish loading.
+  const allRequestors = useMemo(() => {
+    const requestors = [...initialRequestors];
+    if (deferredRequestors) {
+      requestors.push(...deferredRequestors);
+    }
+
+    return requestors;
+  }, [initialRequestors, deferredRequestors]);
+
+  return (
+    <div className="flex flex-col gap-5">
+      <h1>{'List of Set Requests'}</h1>
+
+      <div>
+        <GameAvatar {...game} size={96} />
+      </div>
+
+      <div>{'A set for this game has been requested by the following users:'}</div>
```

Comment:
i18n: Careful, this string is untranslated.

For untranslated strings, you'll only need to worry about updating _lang/en_US.json_ (as you've already done in the PR). Once the PR is merged, those source strings are fed out to the translation team automagically (see [here](https://discord.com/channels/310192285306454017/1299083037908406402/1305281296615866378)).